benchmarks/inference-server/jetstream/jetstream.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: maxengine-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: maxengine-server
+  template:
+    metadata:
+      labels:
+        app: maxengine-server
+    spec:
+      serviceAccountName: benchmark-sa
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x2
+        cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+      containers:
+      - name: maxengine-server
+        image: us-docker.pkg.dev/cloud-tpu-images/inference/maxengine-server:v0.2.0
+        args:
+        - model_name=gemma-7b
+        - tokenizer_path=assets/tokenizer.gemma
+        - per_device_batch_size=4
```

Comment:
We can make the essential configurations from here into Terraform variables as well.

Reply:
Honestly, this is YAGNI to me. Happy to hear an alternatve view. But here is my thinking:

Currently you need a weight transform script per-model for JetStream. So its a pretty bespoke effort to add new models, aka we would need to add a new script and such. 

And mutating a yaml file and running kubectl apply seems equivalent to copying a tfvars file and running terraform apply. We are just cutting out adding extra code until the time when we need it. 

LMKWYT

Reply:
For this PR, I think it's fine to submit as is. Par discussion, to keep things simple and easy to use across the board, plus simplifying the workflow, let's go ahead with tf automation in a follow up PR.